inference-engine/src/mkldnn_plugin/nodes/mkldnn_matrix_nms_node.h

```diff
@@ -82,14 +87,15 @@ class MKLDNNMatrixNmsNode : public MKLDNNNode {
         int64_t classIndex = -1;
         float score = 0.0f;
     };
-    std::string errorPrefix;
-    const std::string inType = "input", outType = "output";
+    std::string m_errorPrefix;
+    const std::string m_inType = "input", m_outType = "output";
```

Comment:
I think add a prefix such as 'm_' is easy to distinguish the class member and local varible. It can not only help to understand the code, but also avoid some errors such as define a same local varible name as the class member which the compiler says nothing.

Anyway if this is forbiden I'll remove all the prefix.

Reply:
lets leave with `m_` prefix